### PR TITLE
Rename from service to processor

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
Case has now been split into a processor (for events) and an API (for RESTful synchronous calls) so we have renamed this project to reflect that.

# What has changed
Renamed git repo, maven artefact ID, docker image.

# How to test?
Make sure Travis build works. Build locally.

# Links
Trello: https://trello.com/c/wj2LhC2Z/642-case-service-a-new-hope-13

# Screenshots (if appropriate):

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,8 +19,7 @@ script: travis_wait mvn verify jacoco:report
 after_success:
 - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
   docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
-  docker tag "$SOURCE_IMAGE_NAME" "$DESTINATION_IMAGE_NAME";
-  docker push "$DESTINATION_IMAGE_NAME";
+  docker push "IMAGE_NAME";
```
Comment (liamtoozer) [inline .travis.yml]: Think `IMAGE_NAME` is just missing a `$`

**Comment by NickGrantONS** [comment]
[Case service: A New Hope (13)](https://trello.com/c/wj2LhC2Z/642-case-service-a-new-hope-13)

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# census-rm-case-service-v2
+# census-rm-case-processor
 Case Service using latest versions of Java, Spring Integration and using DSL instead of XML for configuration.
 
 # How to run
-The new Case Service v2 requires Postgres, Rabbit and the IAC service to be running - you can use census-rm-docker-dev to start these dependencies.
+The new Case Service Processor requires Postgres, Rabbit and the IAC service to be running - you can use census-rm-docker-dev to start these dependencies.
```
Comment (nealedj) [inline README.md]: Case ~Service~ Processor?

**Comment by NickGrantONS** [comment]
disagree that docker-compose should live in the root - it's only for testing

**Comment by AdeHarrison** [comment]
it should be in consistent wherever it lives

**Comment by NickGrantONS** [comment]
let's make it consistently correct then (i.e. putting `application-test.yml` in the `main` folder instead of the `test` folder was always a consistent mistake... which we should fix)

**Pull request merged**